src/ape/api/transactions.py

```diff
@@ -259,3 +282,312 @@ def await_confirmations(self) -> "ReceiptAPI":
                 time.sleep(time_to_sleep)
 
         return self
+
+    def show_trace(self, verbose: bool = False, file: IO[str] = sys.stdout):
+        """
+        Display the complete sequence of contracts and methods called during
+        the transaction.
+
+        Args:
+            verbose (bool): Set to ``True`` to include more information.
+            file (IO[str]): The file to send output to. Defaults to stdout.
+        """
+        tree_factory = CallTraceParser(self, verbose=verbose)
+        call_tree = self.provider.get_call_tree(self.txn_hash)
```

Comment:
What happens if this fails?

Reply:
Ping on this

Reply:
you get:

```
ERROR: (APINotImplementedError) Attempted to call method 'ProviderAPI.get_call_tree', method not supported.
```

Reply:
Providers implement `get_call_tree()` however they can!

`providers` that don't support the `trace_transaction` use the full trace in their implementation of `get_call_tree()`
An example of that is `ape-hardhat`!
https://github.com/ApeWorX/ape-hardhat/pull/71/files#r898447679

Providers that support both implement the full trace still via `get_transaction_trace()` but still use `get_call_tree()` to be the parity trace.
An example of this provider is Alchemy: https://github.com/ApeWorX/ape-alchemy/pull/15/files

Reply:
Providers implement `get_call_tree()` however they can!

`providers` that don't support the `trace_transaction` use the full trace in their implementation of `get_call_tree()`
An example of that is `ape-hardhat`!
https://github.com/ApeWorX/ape-hardhat/pull/71/files#r898447679

Providers that support both implement the full trace still via `get_transaction_trace()` but still use `get_call_tree()` to be the parity trace.
An example of this provider is Alchemy: https://github.com/ApeWorX/ape-alchemy/pull/15/files